mono/metadata/object.c

```diff
@@ -7011,20 +7029,24 @@ mono_wait_handle_get_handle (MonoWaitHandle *handle)
 
 
 static MonoObject*
-mono_runtime_capture_context (MonoDomain *domain)
+mono_runtime_capture_context (MonoDomain *domain, MonoError *error)
 {
 	MONO_REQ_GC_UNSAFE_MODE;
 
 	RuntimeInvokeFunction runtime_invoke;
 
+	mono_error_init (error);
+
 	if (!domain->capture_context_runtime_invoke || !domain->capture_context_method) {
 		MonoMethod *method = mono_get_context_capture_method ();
 		MonoMethod *wrapper;
 		if (!method)
 			return NULL;
```

Comment:
There's no error condition to signal here: `method` could legitimately be `NULL`